ClosedXML_Tests/Excel/Ranges/XLRangeAddressTests.cs

```diff
@@ -289,6 +289,28 @@ public void RelativeRanges()
             Assert.AreEqual("#REF!", rangeAddress.ToString());
         }
 
+        [Test]
+        public void TestSpanProperties()
+        {
+            var ws = new XLWorkbook().AddWorksheet() as XLWorksheet;
+
+            var range = ws.Range("B3:E5");
```

Comment:
Try also `B5:E3` and `E5:B3`. The results must be the same (now they are not). I suggest using `Normalize()` in the implementation.